ui/src/views/Tasks/InteractiveConnect/index.jsx

```diff
@@ -145,6 +149,26 @@ export default class InteractiveConnect extends Component {
         params: { taskId },
       },
     } = this.props;
+    const { notificationNotFired, notificationToggle } = this.state;
+
+    if (
+      // this.getInteractiveStatus === INTERACTIVE_TASK_STATUS.READY &&
+      notificationNotFired &&
+      notificationToggle &&
+      Notification.permission === 'granted'
+    ) {
+      const notification = new Notification(
+        'Your interactive task is ready for connecting. Connect while the task is available.'
+      );
+
+      setTimeout(() => {
+        notification.close();
+      }, 30000);
```

Comment:
Instead of handling that logic here, let's create a notify file in `src/utils/notify.js`.

```js
import logo from '../images/logo.png';

/*
 Display a Desktop notification using the Notifications API to the user.
 */
export default ({ body, icon }) => {
  const notification = new Notification(process.env.APPLICATION_NAME, {
    icon: icon || logo,
    body,
  });

  notification.addEventListener('click', () => {
    notification.close();
    window.focus();
  });

  setTimeout(notification.close, 30000);
};
```